packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -884,6 +891,27 @@ const main = async (opts) => {
     if (interactive) {
         const questions = getQuestions ? getQuestions() : []
         const projectAnswers = await prompt(questions, answers)
+        // Only prompt for plugin selection on interactive presets
+        if (Object.keys(pluginConfig?.plugins || {}).length > 0) {
```

Comment:
We need to move extension selection prompt for `interactive` preset only. Otherwise, pre-defined non-interactive presets will get this prompt.